tools/ci/od_lints.dm

```diff
@@ -0,0 +1,58 @@
+/**
+ * Shiptest OpenDream linting config:
+ * See https://github.com/OpenDreamProject/OpenDream/blob/master/DMCompiler/DMStandard/DefaultPragmaConfig.dm
+ */
+
+// This is the default error/warning/notice/disable setup when the user does not mandate a different file or configuration.
+// If you add a new named error with a code greater than 999, please mark it here.
+
+//1000-1999
+#pragma FileAlreadyIncluded error
+#pragma MissingIncludedFile error
+#pragma InvalidWarningCode error
+#pragma MisplacedDirective error
+#pragma UndefineMissingDirective error
+#pragma DefinedMissingParen error
+#pragma ErrorDirective error
+// Explicitly kept at warning as this is the #warn define.
+#pragma WarningDirective warning
+#pragma MiscapitalizedDirective error
+
+//2000-2999
+#pragma SoftReservedKeyword error
+#pragma DuplicateVariable error
+#pragma DuplicateProcDefinition error
+#pragma PointlessParentCall error
+#pragma PointlessBuiltinCall error
+#pragma SuspiciousMatrixCall error
+#pragma FallbackBuiltinArgument error
+#pragma PointlessScopeOperator error
+#pragma MalformedRange error
+#pragma InvalidRange error
+#pragma InvalidSetStatement error
+#pragma InvalidOverride error
+#pragma InvalidIndexOperation error
+#pragma DanglingVarType error
+#pragma MissingInterpolatedExpression error
+#pragma AmbiguousResourcePath error
+#pragma SuspiciousSwitchCase error
+#pragma PointlessPositionalArgument error
+// NOTE: The next few pragmas are for OpenDream's experimental type checker
+// This feature is still in development, elevating these pragmas outside of local testing is discouraged
+// An RFC to finalize this feature is coming soon(TM)
+// BEGIN TYPEMAKER
+#pragma UnsupportedTypeCheck notice
+#pragma InvalidReturnType notice
+#pragma InvalidVarType notice
+#pragma ImplicitNullType notice
+#pragma LostTypeInfo notice
+// END TYPEMAKER
+#pragma UnimplementedAccess error
+
+//3000-3999
+#pragma EmptyBlock notice
```

Comment:
Since you've already addressed all empty blocks in this PR, this can be set to disabled unless you really want lints popping up for empty blocks. OD has no issues with empty blocks.

Reply:
Huh, interesting. Yeah, I figure I'll disable it then, lol.